### PR TITLE
[DEV-3694] Add "Add your prayer" Button

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?
This PR adds the "Add your prayer" button to the top of the prayer screen when you select an option from the menu. 
### How do I test this PR?
1. Start up your sim.
2. Go to the Prayer tab.
3. Tap on a menu option under the "Ask for prayer" section.
4. The "Ask for prayer" section should disappear and be replaced with an "Add your prayer" button at the top of the screen while also displaying the appropriate content for that tab. 

Before tapping the menu option:
Screenshot 2019-04-28 17 13 01

After tapping the menu option:
Screenshot 2019-04-28 17 13 07

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [DEV-3694](https://newspring.atlassian.net/browse/DEV-3694)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by richarddubay** [comment]
@redreceipt Linting is failing here because of stuff that I'm not sure how to fix in your branch, but the tests pass just fine. I feel like this is in a pretty good place. Let me know if you see otherwise.

**Comment by IsaacHardy** [comment]
@richarddubay Wait, doesn't this have components that exist in @redreceipt PR?

**Comment by richarddubay** [comment]
@IsaacHardy Yep. I forgot to change the base branch to his. Fixed that.

File: packages/newspringchurchapp/src/prayer/index.js [hunk 1]
```diff
@@ -178,7 +213,23 @@ class PrayerMenu extends PureComponent {
...
       <>
-        <AddPrayerCardConnected {...this.props} />
+        {this.state.showAddPrayerCard ? (
+          <StyledAddPrayerContainer>
+            <AddPrayerCardConnected {...this.props} />
+          </StyledAddPrayerContainer>
+        ) : null}
+        {!this.state.showAddPrayerCard ? (
+          <StyledContainer>
+            <NSIcon name={'arrow-up'} fill={this.props.tint} size={24} />
```
Comment (IsaacHardy) [inline packages/newspringchurchapp/src/prayer/index.js]: This should be within the button press, too. After testing, trying to click the arrow is a frustrating experience and doesn't cause the view to pan up.

**Review by IsaacHardy — CHANGES_REQUESTED** [review]
Left a comment, everything else looks good.

**Review by IsaacHardy — APPROVED** [review]
LGTM

**Pull request merged**